using-kubecost/navigating-the-kubecost-ui/teams.md

```diff
@@ -23,79 +22,254 @@ persistentVolume:
 
-### Method 1: Disabling SAML
+## Getting started
 
-You will need to disable your existing SAML configuration temporarily through this tutorial. You will need to turn off SAML authentication Helm flag before creating your team, then turning it back on.
+To manage teams via the Kubecost UI, you must either have admin-level access to Kubecost and create teams through the UI, or configure via Helm. There are three available methods for getting started.
 
-```yaml
-saml:
-  enabled: false
-```
+### Method 1: Enable Teams and login in
 
-Now, access the Teams page, and create your initial admin team. See the [Adding a team](teams.md#adding-a-team) section below for specific instructions for using the UI. Once your admin team has been created, reenable SAML authentication:
+In order to enable teams, enable RBAC for your desired IAM protocol (SAML or OIDC) but do not define any values under the respective `groups[]` list. If groups are defined, this will default the installation to the earlier [simple RBAC](teams.md#rbac-teams-versus-simple-rbac) and not the newer teams RBAC. An example of how to enable the new teams RBAC in Helm when using OIDC is shown below.
 
 ```yaml
-saml:
+oidc:
   enabled: true
   rbac:
-    enabled: false
+    enabled: true
 ```
 
-The members of your team are now able to add new members, or create new teams.
+The first person to log in will be added to a default team with admin-level access. They will then be able to configure teams through the UI on the Teams page.
 
-### Method 2: Disable SAML RBAC while leaving SAML enabled
+### Method 2: Enable SAML/OIDC without Teams, create teams, then enable Teams
 
-This method leaves additional authentication in place while SAML RBAC is disabled.
+This method leaves additional authentication in place while RBAC is disabled.
 
 ```yaml
-saml:
+oidc:
   enabled: true
   rbac:
     enabled: false
 ```
 
-Now, access the Teams page, and create your initial admin team. See the [Adding a team](teams.md#adding-a-team) section below for specific instructions for using the UI. Once your admin team has been created, reenable SAML RBAC authentication:
+Now, access the Teams page, and create your initial admin team. See the [Adding a team](teams.md#adding-a-team) section below for specific instructions for using the UI. Once your admin team has been created, reenable RBAC authentication:
 
 ```yaml
-saml:
+oidc:
   enabled: true
   rbac:
     enabled: true
 ```
 
 The members of your team are now able to add new members, or create new teams.
 
+### Method 3: Configure teams through Helm values
+
+{% hint style="warning" %}
+Configuring teams through Helm will disable configuration of teams through the UI.
+{% endhint %}
+
+
+Kubecost allows teams to be defined in Helm values obviating the need to create them in the UI. An example values snippet is shown below.

```

Comment:
```suggestion
Kubecost allows teams to be defined in Helm values obviating the need to create them in the UI, however the feature must be enabled via Helm. An example values snippet is shown below.
```

Reply:
I reworded this to make is less confusing, and provided an explicit link back up to where the example is.